src/emulator/auth/widget_ui.ts

```diff
@@ -385,16 +396,21 @@ button {
 }
 
 .callout {
-  align-items: center;
   background: #e5eaf0;
   color: #476282;
   display: flex;
   flex-direction: row;
   padding: 12px 24px;
 }
 
+.callout-warning {
+  background: #fff3e0;
+  color: #bf360c;
+}
+
 .callout .content {
   flex: 1;
+  align-self: center;
```

Comment:
nit: consider alphabetizing these. I started doing this style-- we don't need to continue but worth considering

Reply:
Next time I touch this file I'll sort them all